.github/actions/snippets-tests/action.yml

```diff
@@ -3,6 +3,10 @@ description: Run Snippets Testss
 runs:
   using: "composite"
   steps:
+    - uses: actions/setup-java@v1
+      with:
+        java-version: "11"
```

Comment:
It should be putted somewhere that we are upgrading to Java 11, because people that are using Java 8 wont be able to use the SDK.

Also it is possible to make HTTP Requests using Java 8